include/internal/catch_config.hpp

```diff
@@ -52,6 +52,7 @@ namespace Catch {
         Verbosity verbosity = Verbosity::Normal;
         WarnAbout::What warnings = WarnAbout::Nothing;
         ShowDurations::OrNot showDurations = ShowDurations::DefaultForReporter;
+        double minDuration = -1;
```

Comment:
I think having `0.` mean "turned off" is fine here. If an user wants all durations to be shown, they can set up `showDurations::Always`.

Reply:
I'm always leery of using a value that might possibly have been intended as valid for a sentinel value.  You never know when there might be extra layers between the user and the value (e.g. a GUI or a wrapper script) which makes it less easy to pass other arguments.